apps/web/src/modules/create-dao/components/Artwork/ArtworkForm.tsx

```diff
@@ -49,73 +50,81 @@ export const Artwork: React.FC<ArtworkProps> = ({ title }) => {
-            onBlur={formik.handleBlur}
-            errorMessage={
-              formik.touched?.projectDescription && formik.errors?.projectDescription
-                ? formik.errors?.projectDescription
-                : undefined
-            }
-            placeholder={'Nouns is an experiment which combines...'}
-          />
-
-          <ArtworkUpload
-            {...formik.getFieldProps('artwork')}
-            inputLabel={'Artwork'}
-            formik={formik}
-            id={'artwork'}
-            onChange={formik.handleChange}
-            onBlur={formik.handleBlur}
-            helperText={
-              'Builder uses folder hierarchy to organize your assets. Upload a single folder containing a subfolder for each trait. Each subfolder should contain every variant for that trait.'
-            }
-            errorMessage={
-              formik.touched.artwork && formik.errors?.artwork
-                ? formik.errors?.artwork
-                : undefined
-            }
-          />
+      {(formik) => {
+        return (
+          <Form>
+            <TextArea
+              {...formik.getFieldProps('projectDescription')}
+              inputLabel={'Collection Description'}
+              formik={formik}
+              id={'projectDescription'}
+              onChange={(e) => {
+                formik.handleChange(e)
+                setSetUpArtwork({
+                  ...setUpArtwork,
+                  projectDescription: (e.target as HTMLInputElement).value,
+                })
+              }}
+              onBlur={formik.handleBlur}
+              errorMessage={
+                formik.touched?.projectDescription && formik.errors?.projectDescription
+                  ? formik.errors?.projectDescription
+                  : undefined
+              }
+              placeholder={'Nouns is an experiment which combines...'}
+            />
 
-          <Flex justify={'space-between'} mt={'x8'}>
-            <Button
-              justify={'center'}
-              align={'center'}
-              borderRadius={'curved'}
-              h={'x15'}
-              minH={'x15'}
-              minW={'x15'}
-              variant={'secondary'}
-              onClick={handlePrevious}
-              aria-label="Back"
-            >
-              <Icon id="arrowLeft" />
-            </Button>
-            <Button
-              flex={1}
-              borderRadius={'curved'}
-              width={'auto'}
-              ml={'x2'}
-              minH={'x15'}
-              type="submit"
-              disabled={
-                !isEmpty(formik.errors) ||
-                formik.isSubmitting ||
-                isUploadingToIPFS ||
-                ipfsUpload.length === 0
+            <ArtworkUpload
+              {...formik.getFieldProps('artwork')}
+              inputLabel={'Artwork'}
+              formik={formik}
+              id={'artwork'}
+              onChange={formik.handleChange}
```

Comment:
We should be sure to update the store on submit for this field as well -- in the current build you cannot reupload artwork (which has been a known bug) but this PR reveals the issue, would be great to fix this as well

Reply:
Just took a little dive on this one. 

There's definitely the issue with not being to upload properly, but it looks like there's another issue with the preview (it still sticks to the old artwork preview, even if I upload new files) , which I suspect is why the formValues weren't being to the store in the first place. 

Since there are users who are having issues with the description rn, would it be okay if we get this fix in and then fix the Artwork issue on a separate PR? Seems like there's a lot going on under the hood that I'll need to wade through.

Reply:
Just tried again and the entire app crashes for me.